tools/cvm-image-rewriter/README.md

```diff
@@ -94,44 +128,73 @@ steps:
     virsh net-start default
     ```
 
+### 3.2 Run the tool
+
+The tool provides several plugins to customize the initial image. It will generate an `output.qcow2` under current directory.
+
+Before running the tool, please choose the plugins that are needed.You can skip any plugin by creating a file "NOT_RUN" under the plugin directory.
+For example:
+
+    ```
+    touch pre-stage/01-resize-image/NOT_RUN
+    ```
+
+If the guest image is used for CCNP deployment, it's recommended to run below plugin combination according to different initial guest image type.
+Others are not required by CCNP and can be skipped.
+|  Base image | 01  | 02  | 03  | 04  | 05 | 07 | 08 | 09 | 60 | 98 |
+|---|---|---|---|---|---|---|---|---|---|---|
+|  Ubuntu base image | | | | | Y| | Y| Y| | |
+| TD enlightened image | | | | | | | Y| Y| | |
 
-### 2.2 Customize
+**NOTE:**
+  - TD enlightened image means the image already has TDX kernel. If not, plugin 05 is needed to install TDX kernel.
+  - Plugin 08 and 09 prepares device permission for CCNP deployment.
+  - Other plugins are optional for CCNP deployment. 
 
+The tool supports parameters as below.
 ```
-$ ./run.sh -h
```

Comment:
The option `-h` exists in the function `process_args` of the script `run.sh`, but is missing in message printed from the function `usage`. We can update function `usage`, instead of removing the option, which may make user confuse.

Reply:
Added -h in usage function.

Reply:
@HaokunX-intel Updated. Please review it again.

Reply:
LGTM